knowledge-base/how-to-dynamically-select-and-filter-data-in-blazor.md

```diff
@@ -17,18 +17,34 @@ res_type: kb
 			<td>Product</td>
 			<td>Progress® Telerik® Reporting</td>
 		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>Blazor Report Viewer</td>
+		</tr>
 	</tbody>
 </table>
 
 
 ## Description
-The Telerik Report Viewer makes it easy to let your user select the report they want to see and filter the data in that report to get the information they need. Here’s how to make that happen in a Blazor app.
+The Telerik Report Viewer makes it easy to let your user select and filter the data of a report to get only the information they need. Out of the box, we implement the report parameter controls in the report viewer which allow you to do this, however, there are times where one may wish to use controls outside of the report viewer widget since it supports only a limited type of report parameter controls out of the box.
+
+For this example, we will use the **Invoice** report that we ship with installation of the Telerik Reporting product. That report, and the other example reports, can be found in the installation directory. For example:
```

Comment:
I suggest writing this on a single line, as an inline code snippet (see the Wiki):

...For example `{install_dir}/Report Designer/Examples`.

Additionally, instead of using bold text for the **Invoice**, you may put it also in an inline code:
.....`Invoice`....
I think this makes it more easily distinguishable.
The same is valid for the other bold text inline, like file names, tag names, etc.